src/internal/general.ts

```diff
@@ -130,3 +131,24 @@ export async function queryIndexer<T>(args: {
   });
   return data;
 }
+
+export async function getProcessorStatuses(args: { aptosConfig: AptosConfig }): Promise<GetProcessorStatusResponse> {
+  const { aptosConfig } = args;
+
+  const graphqlQuery = {
+    query: GetProcessorStatus,
+  };
+
+  const data = await queryIndexer<GetProcessorStatusQuery>({
+    aptosConfig,
+    query: graphqlQuery,
+    originMethod: "getIndexerLastSuccessVersion",
```

Comment:
this should be getProcessorStatuses i think

Reply:
Good catch.